database/src/main/kotlin/no/nav/su/se/bakover/database/grunnlag/FormuegrunnlagPostgresRepo.kt

```diff
@@ -21,7 +22,7 @@ internal class FormuegrunnlagPostgresRepo(
     private val dbMetrics: DbMetrics,
 ) {
     internal fun lagreFormuegrunnlag(
-        behandlingId: UUID,
+        behandlingId: BehandlingsId,
```

Comment:
Merker meg at vi fremdeles har en `import java.util.UUID`, burde kanskje ikke trenge den importen lenger? Gjelder for tilsvarende klasser også.

Reply:
Grunnlagene benytter seg av egne Ider som er UUID